crates/autopilot/src/database/auction.rs

```diff
@@ -84,6 +85,7 @@ impl Postgres {
                 })
                 .try_collect()
                 .await?;
+        info!("all solvable orders fetched in {}", Utc::now() - start);
```

Comment:
Usually we try to avoid baking dynamic values into log strings. The reason is that the part before or after the dynamic value can be ambiguous which would then require a query like `"string before value" and "string after value"`. This obviously doesn't apply here but let's stick to this to keep the code consistent.
```suggestion
        info!(time = Utc::now() - start, "fetched all solvable orders");
```